src/components/SlateEditor/plugins/definitionList/handlers/onEnter.ts

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { Editor, Element, Path, Transforms, Node, Range, Point } from 'slate';
+import { TYPE_DEFINTION_TERM, TYPE_DEFINTION_DESCRIPTION } from '../types';
+import { definitionTerm } from '../utils/defaultBlocks';
+
+const onEnter = (
+  editor: Editor,
+  e: KeyboardEvent,
+  nextOnKeyDown: ((e: KeyboardEvent) => void) | undefined,
+) => {
+  if (e.shiftKey && nextOnKeyDown) return nextOnKeyDown(e);
+  if (!editor.selection && nextOnKeyDown) return nextOnKeyDown(e);
+  else if (!editor.selection) return undefined;
+
+  const [selectedDefinitionItem, selectedDefinitionItemPath] = Editor.parent(
+    editor,
+    editor.selection.anchor.path,
+  );
+
+  if (!selectedDefinitionItem) {
+    return nextOnKeyDown && nextOnKeyDown(e);
+  }
+
+  if (
+    Element.isElement(selectedDefinitionItem) &&
+    selectedDefinitionItem.type === TYPE_DEFINTION_TERM
+  ) {
+    const [term, termPath] = [selectedDefinitionItem, selectedDefinitionItemPath];
+    const maybeDescription = Editor.next(editor, { at: selectedDefinitionItemPath });
```

Comment:
Regner med at dette bare er for å få kortere variabelnavn? Hvis ja, så burde du gå over til å bare bruke de.